packages/components/.npmignore

```diff
@@ -0,0 +1,7 @@
+.storybook/
+src/
+.babelrc
+.eslintrc
```

Comment:
mocha config can be ignored too. everything that we don't need to import this package can/should be ignored